src/ollie-bird/game-object/Baddie.ts

```diff
@@ -62,22 +62,26 @@ export default class Baddie extends GameObject {
 	}
 
 	protected override render(context: CanvasRenderingContext2D) {
-		using _ = this.transform.push(context);
+		const disposable = this.transform.push(context);
```

Comment:
Since we worked out that this is due to an old verison of chrome, please undo this change.